blockchain/chainio.go

```diff
@@ -479,8 +482,13 @@ func serializeSpendJournalEntry(stxos []spentTxOut) ([]byte, error) {
 	var size int
 	var sizes []int
 	for i := range stxos {
-		// size += spentTxOutSerializeSize(&stxos[i])
 		sz := spentTxOutSerializeSize(&stxos[i])
+		if sz < minimumStxoSize {
```

Comment:
I think spentTxOutSerializeSize should check this and return error so caller doesn't forget.